### DISCUSSION TITLE
how to get photo file data ([]byte)

### DISCUSSION THREAD

**Discussion by hdbjlizhe** [gotd/td#1277] [Q&A]
code as follows，how can I get the photo file data here. I want to store the photo to path named xxx.jpg，thanks a lot
````
dispatcher.OnNewMessage(func(ctx context.Context, e tg.Entities, update *tg.UpdateNewMessage) error {
   if msg,ok:=update.Message.(*tg.Message);ok{
      switch media := update.Message.(*tg.Message).Media.(type) {
      case *tg.MessageMediaPhoto: 
      //how can I get the photo file data here. I want to store the photo to path named xxx.jpg
      }
   }
}
````

**Comment by lsy88** [comment]
> 代码如下，这里怎么获取照片文件的数据啊，我想把照片存放到xxx.jpg这个路径下，谢谢
> 
> ```
> dispatcher.OnNewMessage(func(ctx context.Context, e tg.Entities, update *tg.UpdateNewMessage) error {
>    if msg,ok:=update.Message.(*tg.Message);ok{
>       switch media := update.Message.(*tg.Message).Media.(type) {
>       case *tg.MessageMediaPhoto: 
>       //how can I get the photo file data here. I want to store the photo to path named xxx.jpg
>       }
>    }
> }
> ```

Do you have a solution?
